cfme/fixtures/service_fixtures.py

```diff
@@ -23,7 +23,7 @@ def dialog():
                      tab_label="tab_" + fauxfactory.gen_alphanumeric(), tab_desc="my tab desc",
                      box_label="box_" + fauxfactory.gen_alphanumeric(), box_desc="my box desc")
     service_dialog.create(element_data)
-    return dialog
+    yield service_dialog
```

Comment:
OPTIONAL: it makes sense to add teardown/destructor part if you decided to use yield_fixture